layouts/_partials/scripts/jstoc.html

```diff
@@ -0,0 +1,44 @@
+{{ $version := .Site.Params.jstoc.version | default "latest" -}}
+
+{{ $css_url := printf "https://cdn.jsdelivr.net/npm/tocbot@%s/dist/tocbot%s.css" $version (cond hugo.IsProduction ".min" "") -}}
```

Comment:
Refactor the code to define `$resourceBaseName` as `tocbot%s` using `(cond hugo.IsProduction ".min" "")`. Note that this variable will be used in four places.